spec/rubocop/cop/performance/redundant_merge_spec.rb

```diff
@@ -3,8 +3,12 @@
 
 require 'spec_helper'
 
-describe RuboCop::Cop::Performance::RedundantMerge do
-  subject(:cop) { described_class.new }
+describe RuboCop::Cop::Performance::RedundantMerge, :config do
+  subject(:cop) { described_class.new(config) }
+
+  let(:cop_config) do
+    { 'MaxKeyValuePairs' => 2 }
```

Comment:
Actually it's not, it's `nil` if I don't define it here (otherwise I wouldn't have thought to cast it `to_i`). The cop only sees a completely empty hash for `cop_config` unless I pass something, I don't know if I'm setting it up incorrectly on lines 6/7. But like you say it is resilient to config changes so maybe it doesn't matter.